src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/EndBusinessProcessTaskHandler.java

```diff
@@ -23,21 +24,27 @@
 @RequiredArgsConstructor
 public class EndBusinessProcessTaskHandler implements BaseExternalTaskHandler {
 
+    public static final String NOT_RETRYABLE_MESSAGE = "Stopping multiple calls, END_BUSINESS_PROCESS already performed.";
     private final CoreCaseDataService coreCaseDataService;
     private final CaseDetailsConverter caseDetailsConverter;
     private final ObjectMapper mapper;
 
     @Override
     public void handleTask(ExternalTask externalTask) {
-        ExternalTaskInput externalTaskInput = mapper.convertValue(externalTask.getAllVariables(),
-                                                                  ExternalTaskInput.class);
+        ExternalTaskInput externalTaskInput = mapper.convertValue(
+            externalTask.getAllVariables(),
+            ExternalTaskInput.class
+        );
         String caseId = externalTaskInput.getCaseId();
-
         StartEventResponse startEventResponse = coreCaseDataService.startUpdate(caseId, END_BUSINESS_PROCESS);
         CaseData data = caseDetailsConverter.toCaseData(startEventResponse.getCaseDetails());
         BusinessProcess businessProcess = data.getBusinessProcess();
-
-        coreCaseDataService.submitUpdate(caseId, caseDataContent(startEventResponse, businessProcess));
+        switch (businessProcess.getStatusOrDefault()) {
```

Comment:
Here coreCaseDataService.startUpdate is started then we should then call submitUpdate.
Whats point of raising exception here? I think it doesn't seems to be correct